Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -92,6 +92,15 @@
     *acts.examples.itk.itkSeedingAlgConfig(
         acts.examples.itk.InputSpacePointsType.PixelSpacePoints
     ),
+    initialSigmas=[
+        1 * u.mm,
+        1 * u.mm,
+        1 * u.degree,
+        1 * u.degree,
+        0.1 / u.GeV,
+        1 * u.ns,
+    ],
```

Comment:
These are just the default values, right? I guess you put this here such that it can be changed more easily later on?

Reply:
yes and also to make clear what is used. I think default configs for such things are just evil

Reply:
agree 100%